### PR TITLE
CODENVY-522 Remove roles context

### PR DESCRIPTION
**Pull request by skabashnyuk**
In 3.x we have roles depending on workspace and account id of request. Since we are moving to permission based authorisation -we no longer need it.

### CODE REVIEW

File: core/codenvy-hosted-sso-client/src/main/java/com/codenvy/auth/sso/client/ServerClient.java [hunk 1]
```diff
@@ -29,13 +29,10 @@
      *         - sso authentication token.
      * @param clientUrl
      *         - url of client who asking the principal.
-     * @param workspaceId
-     *         - id of workspace for which requested roles.
-     * @param accountId
-     *         - id of account for which requested roles.
      * @return - principal with roles. If token is not valid return null.
+     * @Depricated use ServerClient.getUser(String token, String clientUrl)
```
Comment (sleshchenko) [inline core/codenvy-hosted-sso-client/src/main/java/com/codenvy/auth/sso/client/ServerClient.java]: @Deprecated ?

**Comment by sleshchenko** [comment]
LGTM

**Comment by benoitf** [comment]
Could you please explain why it's no longer used and why there is no JIRA issue related to this commit

**Comment by benoitf** [comment]
thanks

**Comment by skabashnyuk** [comment]
np

**Comment by vparfonov** [comment]
ok

**Pull request merged**